cypress/e2e/bulk-edit/csv/bulk-edit-csv-drag-and-drop-is-enabled-after-file-is-uploaded.cy.js

```diff
@@ -0,0 +1,51 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+
+describe('bulk-edit', () => {
```

Comment:
add another nested describe with text 'csv approach', see any other test in this folder (e2e/bulk-edit/csv)

Reply:
added